libraries/eventformatter/impl/src/main/kotlin/io/element/android/libraries/eventformatter/impl/DefaultTimelineEventFormatter.kt

```diff
@@ -52,13 +52,11 @@ class DefaultTimelineEventFormatter @Inject constructor(
             is StateContent -> {
                 stateContentFormatter.format(content, senderDisambiguatedDisplayName, isOutgoing, RenderingMode.Timeline)
             }
-            is LegacyCallInviteContent -> {
-                sp.getString(CommonStrings.common_call_invite)
-            }
             is CallNotifyContent -> {
                 sp.getString(CommonStrings.common_call_started)
             }
             RedactedContent,
+            is LegacyCallInviteContent,
```

Comment:
The case `is CallNotifyContent` should probably be moved here as well, but this is a bit out of scope of this PR.